packages/gafl-webapp-service/src/test-utils.js

```diff
@@ -0,0 +1,38 @@
+import { createServer, init, server } from './server.js'
```

Comment:
Is this for unit tests only?  If so can we move to __tests__ folder in the gafl-webapp-service package root?

Reply:
i tried it - it thinks its a test and tries to run it

Reply:
Test suite failed to run

    Your test suite must contain at least one test.

      at onResult (../../node_modules/@jest/core/build/TestScheduler.js:173:18)
          at async Promise.all (index 23)
      at async TestScheduler.scheduleTests (../../node_modules/@jest/core/build/TestScheduler.js:260:11)
      at async runJest (../../node_modules/@jest/core/build/runJest.js:414:19)
      at async _run (../../node_modules/@jest/core/build/cli/index.js:371:7)
      at async runCLI (../../node_modules/@jest/core/build/cli/index.js:224:3)

I have moved it to /tests